tests/fast-integration/Makefile

```diff
@@ -57,5 +57,24 @@ ci-application-connectivity-2-compass:
 
 .PHONY: ci-test-eventing
 ci-test-eventing:
-	npm ci
-	npm run test-eventing
+	npm install
+	npm run eventing-upgrade-test-prep
```

Comment:
There is no actual upgrade happening in the FI tests. So I suggest to use the following for the npm targets:

- test-eventing-setup
- test-eventing-run
- test-eventing-cleanup